lib/dirList.js

```diff
@@ -159,12 +160,9 @@ const dirList = {
    * @return {boolean}
```

Comment:
```suggestion
   * @return {boolean | undefined}
```

Changing signatures could be a breaking change, so you could also do this:
```js
return (options.names?.includes(path.basename(route)) ||
      (options.names?.includes('/') && route[route.length - 1] === '/') ?? false)
```